docs/general/attacks.md

```diff
@@ -0,0 +1,81 @@
+---
+id: attacks
+title: Protection from Attacks
```

Comment:
The title is too generic.

Earlier, this section was under "Verifying Extrinsic/Transactions," so it was implicit that these were transaction-related attacks. 

Some titles to consider - 

"Example Attack Scenarios"
"Transaction Verification"
"Why Verify Transactions?"